backend/src/yeomanui.ts

```diff
@@ -152,24 +152,17 @@ export class YeomanUI {
            https://yeoman.github.io/generator/Generator.html#run
          ... but .d.ts hasn't been updated for a while:
            https://www.npmjs.com/package/@types/yeoman-generator */
-      this.gen.run((err) => {
-        let message: string;
-        const destinationRoot = this.gen.destinationRoot();
-        if (err) {
-          message = `${generatorName} generator failed.\n\n${this.getErrorInfo(err)}`;
-          this.logError(err, message);
-          this.youiEvents.doGeneratorDone(false, message, destinationRoot);
-        } else {
-          message = `The '${generatorName}' project has been generated.`;
-          this.logger.debug("done running yeomanui! " + message + ` You can find it at ${destinationRoot}`);
-          this.youiEvents.doGeneratorDone(true, message, destinationRoot);
-        }
+      const destinationRoot = this.gen.destinationRoot();
```

Comment:
Must get the destRoot after generator has been run.
The destRoot might be changed during run.

Reply:
Done